cypress/utils/commands.js

```diff
@@ -10,13 +10,37 @@
 import { BASE_PATH } from './constants';
 import { TestFixtureHandler } from '../lib/test_fixture_handler';
 
+/**
+ * Send a request to OpenSearch
+ *
+ * @param {Partial<Cypress.RequestOptions>} requestOptions
+ */
+
+Cypress.Commands.add('sendOpenSearch', (requestOptions) => {
```

Comment:
i think `requestOpenSearch` is better to conform to cypress's naming of `cy.request()`